migration/migrate.go

```diff
@@ -0,0 +1,45 @@
+package migration
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-devs-ua/octagon/app/repository/pg"
+)
+
+// Execute migrations.
+// Use "migrate-up" argument after run of program. It will create new table
+// Or use "migrate-down" - that will remove table
+// Example: go run main.go migrate-up
+func Migrate(repo *pg.Repo) error {
```

Comment:
It's better to have migrations as a separate component. 
It shall accept DB connection and do all the magic.

In theory we still can accept repository but in the future we will want to move migrations as far from the main app logic as possible. That's why i would not rely on repository but just on DB conection.

Reply:
[fix: Call connection in main](https://github.com/go-devs-ua/octagon/pull/4/commits/1a77b437f50bc589577f71dbfbc25b553eb18e63)

Reply:
fix: change in migration from store to simple db connection